app/src/main/java/no/usn/mob3000/ui/screens/user/ProfileScreen.kt

```diff
@@ -115,13 +184,26 @@ fun ProfileHeader() {
                 .clip(CircleShape)
                 .border(2.dp, Color.Black, CircleShape)
         )
-
-        Text(
-            text = "Example user",
-            fontWeight = FontWeight.Bold,
-            fontSize = 18.sp,
-            modifier = Modifier.padding(top = 8.dp)
-        )
+        userResult.onSuccess { user ->
+            user?.let {
+                Text(
+                    text = it.displayName,
+                    fontWeight = FontWeight.Bold,
+                    fontSize = 18.sp,
+                    modifier = Modifier.padding(top = 8.dp)
+                )
+            } ?: Text(
+                text = "No user data",
```

Comment:
Composable strings should be string resources, not directly used. It impedes translations.